content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: A New Istiod Deployment Model
```

Comment:
Consider ... "An Alternative Istiod Deployment Model".

The title could be interpreted as if the current deployment model is changing.

Reply:
Good point, Updated, thanks Dan!